src/CommandLoop.hs

```diff
@@ -178,17 +180,25 @@ configSession state clientSend config = do
     handleGhcError :: GHC.GhcException -> GHC.Ghc String
     handleGhcError e = return $ GHC.showGhcException e ""
 
-runCommand :: IORef State -> ClientSend -> Command -> GHC.Ghc ()
-runCommand _ clientSend (CmdCheck file) = do
+runCommand :: IORef State -> ClientSend -> Bool -> Command -> GHC.Ghc ()
```

Comment:
I'd rather pass the whole `Config` into `runCommand` than only the TH flag. This documents a bit better and aligns the signature more with `configSession`.